src/main/java/homework_1/Main.java

```diff
@@ -3,7 +3,94 @@
 public class Main {
 
   public static void main(String[] args) {
-    System.out.println("Hello homework!");
+    //Домашняя работа от 05.07.2021
+    if (args == null || args.length < 1) {
+      return;
+    }
+
+    for (String arg : args) {
+      if (arg.equals("error")) {
+        System.out.println(ConsoleColors.RED_BOLD + "Alarm!" + ConsoleColors.RESET);
+        break;
+      } else {
+        System.out.println(arg + ": " + arg.length() + " letters");
+      }
+    }
+  }
+
+  public class ConsoleColors {
```

Comment:
Plz remove this class and add only the constants you need to initial class

Reply:
Required changes are done.